keras_cv/metrics/object_detection/box_coco_metrics_test.py

```diff
@@ -177,3 +177,43 @@ def test_coco_metric_suite_ragged_prediction(self):
             # The metrics will be all 1.0 because the prediction and ground
             # truth is identical.
             self.assertAllEqual(metrics[metric], 1.0)
+
+    def test_coco_metric_suite_ragged_inputs(self):
+        bounding_boxes = {
+            "boxes": tf.ragged.constant(
+                [
+                    [[10, 10, 20, 20], [100, 100, 150, 150]],  # small, medium
+                    [
+                        [200, 200, 400, 400],  # large
+                    ],
+                ],
+                dtype=tf.float32,
+            ),
+            "classes": tf.ragged.constant(
+                [
+                    [0, 1],
+                    [2],
+                ],
+                dtype=tf.float32,
+            ),
+            "confidence": tf.ragged.constant(
+                [
+                    [0.7, 0.8],
+                    [0.9],
+                ],
+                dtype=tf.float32,
+            ),
+        }
+        dense_bounding_boxes = bounding_box.to_dense(bounding_boxes)
+        ragged_bounding_boxes = bounding_box.to_ragged(dense_bounding_boxes)
```

Comment:
By round-tripping ragged-dense-ragged, we're depending on the behavior of this round trip in a way that I think we shouldn't need to for this test -- can we just say `ragged_bounding_boxes=bounding_boxes` (or just rename `bounding_boxes` to `ragged_bounding_boxes` above?

Reply:
I have updated the unit test by utilizing golden samples.